apps/mull-ui/src/app/components/subnavigation-bar/subnavigation-bar.tsx

```diff
@@ -0,0 +1,39 @@
+import { ROUTES } from '../../../constants';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import './subnavigation-bar.scss';
+
+export const SubNavigationBar = () => {
```

Comment:
Add this to the `apps/mull-ui/src/app/components/index.ts` 👀

Reply:
you do a good job of catching these minor details!

Reply:
More 👀 more 💵